pages/securing.rst

```diff
@@ -4,6 +4,12 @@
 Securing Graylog
 ****************
 
+To secure your Graylog Setup, you should not use one of our pre-configured Images, create your own unique installation where you understand each component and secure the environment by design. Expose only the Services that are needed and secure them whenever possible with TLS/SSL and some kind of authentification. Do not use the pre-created appliances as critical production environments. 
+
+The Graylog OVA by default leave MongoDB and Elasticsearch listening to the external interface, just because this is created as a showcase to simplify the creation of a cluster. Never run this in an insecure network. 
```

Comment:
* On the Graylog appliances MongoDB and Elasticsearch is listening on the external interface. This makes the creation of a cluster easier and demonstrates the way Graylog works.